harness/determined/cli/task.py

```diff
@@ -6,16 +6,25 @@
 from determined import cli
 from determined.cli import command, render
 from determined.common import api
-from determined.common.api import authentication
+from determined.common.api import authentication, bindings
 from determined.common.declarative_argparse import Arg, Cmd, Group
 
 
 def render_tasks(args: Namespace, tasks: Dict[str, Dict[str, Any]]) -> None:
+    """
+    Render tasks for JSON, tabulate or csv output.
+
+    The output of endpoint /api/v1/tasks and bindings.get_GetTasks() has an
+    extra dict layer of "allocationIdToSummary". It is an artifact of the
+    limitations of protobuf. This fuction only accepts the value of
+    "allocationIdToSummary" for the sake of better output and cleaner code.
```

Comment:
The whole point of what I was saying is that nothing needs to know about the layer of mapping with `allocationIdToSummary` except places that directly deal with the API; mentioning it here just drags it up somewhere it doesn't need to be. The comment should simply describe the input in its own right: a map from allocation IDs to dicts describing individual tasks.